pkgs/development/libraries/sdformat/default.nix

```diff
@@ -1,12 +1,12 @@
-{ stdenv, fetchurl, cmake, boost, ruby_1_9, ignition, tinyxml
+{ stdenv, fetchurl, cmake, boost, ruby_2_2, ignition, tinyxml
   , name ? "sdformat-${version}"
   , version ? "4.0.0" # versions known to work with this expression include 3.7.0
   , srchash-sha256 ? "b0f94bb40b0d83e35ff250a7916fdfd6df5cdc1e60c47bc53dd2da5e2378163e"
   , ...
   }:
 
 let 
-  ruby = ruby_1_9;
```

Comment:
I bet there is a reason why it had been set to ruby 1.9. Did you test this library?